omego/env.py

```diff
@@ -119,13 +119,16 @@ def __init__(self, parser):
         Add(group, "build",
             "http://%(ci)s/job/%(branch)s/lastSuccessfulBuild/",
             help="Full url of the Jenkins build containing the artifacts")
-        Add(group, "labels", "ICE=3.5",
+        Add(group, "labels", "",
             help="Comma separated list of labels for matrix builds")
 
         Add(group, "downloadurl",
             "http://downloads.openmicroscopy.org",
             help="Base URL of the downloads server")
 
+        Add(group, "ice",
+            "", help="Ice version, default is the latest (release only)")
```

Comment:
Yes, for **release downloads** it'll download the latest iceXX, for CI builds it doesn't, since the assumption is you know what you're doing.